src/main/java/org/jboss/modules/JarFileResourceLoader.java

```diff
@@ -137,15 +137,14 @@ public synchronized ClassSpec getClassSpec(final String fileName) throws IOExcep
             // size unknown
             spec.setBytes(getClassBytes(entry));
             spec.setCodeSource(createCodeSource(entry));
-            return spec;
```

Comment:
These kinds of simplifications are highly subjective/opinion-based.  In general it's a better practice to leave code as is when there is an established style (even if you disagree with it) unless there is some objective reason (like improved performance).  One danger is that you may get into a pointless "commit war" with someone else whose opinions are just as strong (but opposing).